docs/assets/scss/_footer.scss

```diff
@@ -6,12 +6,12 @@
   padding: 4rem 0;
   margin-top: 4rem;
   font-size: 85%;
-  background-color: #f7f7f7;
+  background-color: $gray-100;
```

Comment:
Properties should be ordered padding, margin-top, font-size, text-align, background-color